core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -72,7 +72,7 @@ trait RTS {
    *
    * FIXME: Replace this entirely with the new scheme.
    */
-  final val YieldMaxOpCount = 1048576
+  final val YieldMaxOpCount = 1024
```

Comment:
Could you unfinal YieldMaxOpCount while you're at it?

Reply:
@kaishh makes sense, will do 👍